src/components/scenes/MigrateWalletCompletionScene.tsx

```diff
@@ -0,0 +1,253 @@
+        await newWallet.changeEnabledTokenIds(tokenItems.map(pair => pair.tokenId))
+
+        // Send tokens
+        let feeTotal = '0'
+        const hasError = false
+        const pendingTxs: EdgeTransaction[] = []
+        for (const item of tokenItems) {
+          let tokenSpendInfo: EdgeSpendInfo = {
+            currencyCode: item.currencyCode,
+            spendTargets: [{ publicAddress: newPublicAddress }],
+            pendingTxs,
+            networkFeeOption: 'standard'
+          }
+          try {
+            const maxAmount = await oldWallet.getMaxSpendable(tokenSpendInfo)
+            tokenSpendInfo = { ...tokenSpendInfo, spendTargets: [{ ...tokenSpendInfo.spendTargets[0], nativeAmount: maxAmount }] }
+            const tx = await makeSpendSignAndBroadcast(oldWallet, tokenSpendInfo)
+            const txFee = tx.parentNetworkFee ?? tx.networkFee
+            feeTotal = add(feeTotal, txFee)
+            pendingTxs.push(tx)
+
+            handleItemStatus(item, 'complete')
+          } catch (e: any) {
+            handleItemStatus(item, 'error')
+          }
+        }
+
+        if (!hasError) {
+          // Send mainnet
+          let spendInfo: EdgeSpendInfo = {
+            currencyCode: oldWallet.currencyInfo.currencyCode,
+            spendTargets: [{ publicAddress: newPublicAddress }],
+            pendingTxs,
+            networkFeeOption: 'standard'
+          }
+          try {
+            const maxAmount = await oldWallet.getMaxSpendable(spendInfo)
+            spendInfo = { ...spendInfo, spendTargets: [{ ...spendInfo.spendTargets[0], nativeAmount: maxAmount }] }
+            const amountToSend = sub(maxAmount, feeTotal)
+            spendInfo = { ...spendInfo, spendTargets: [{ ...spendInfo.spendTargets[0], nativeAmount: amountToSend }] }
+            await makeSpendSignAndBroadcast(oldWallet, spendInfo)
+            handleItemStatus(mainnetItem, 'complete')
+          } catch (e) {
+            showError(e)
+            handleItemStatus(mainnetItem, 'error')
+          }
+
+          // const status = await sendTransaction(oldWallet, spendInfo)
+        } else {
+          handleItemStatus(mainnetItem, 'error')
+        }
+      }
+      migrationPromises.push(createNewWalletPromise)
+    }
+
+    for (const migration of migrationPromises) {
+      await migration()
+    }
+
+    setDone(true)
+    return () => {}
+  }, [])
+
+  const renderStatus = useHandler((item: MigrateWalletItem) => {
+    let icon = <ActivityIndicator style={{ paddingRight: theme.rem(0.3125) }} color={theme.iconTappable} />
+    if (itemStatus[item.key] === 'complete') icon = <IonIcon name="checkmark-circle-outline" size={theme.rem(1.5)} color={theme.iconTappable} />
+    if (itemStatus[item.key] === 'error')
+      icon = <IonIcon name="warning-outline" style={{ paddingRight: theme.rem(0.0625) }} size={theme.rem(1.5)} color={theme.dangerText} />
+    return icon
+  })
+
+  const renderRow = useHandler(({ item }) => {
```

Comment:
On another scene, you had `useHandler((data: ListRenderItemInfo<MigrateWalletItem>) => {`. Would that work here?